htdocs/product/list.php

```diff
@@ -237,7 +237,7 @@
 		} else {
 			$labelp = $langs->trans("SellingPrice")." ".$i;
 		}
-		$arrayfields['p.sellprice'.$i] = array('label'=>$labelp, 'checked'=>1, 'enabled'=>$conf->global->PRODUIT_MULTIPRICES, 'position'=>40);
+		$arrayfields['p.sellprice'.$i] = array('label'=>$labelp, 'checked'=>1, 'enabled'=>$conf->global->PRODUIT_MULTIPRICES, 'position'=>'40.'.$i);
```

Comment:
What if there is 11 fields.
position 40.11 will be before 40.2
May be we can make something like
float('40.'.sprintf('%03s',$i));

Reply:
@eldy yes you right, it's fixed, thanks